azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -98,11 +103,6 @@ func resourceArmKubernetesCluster() *schema.Resource {
 							Computed: true,
 						},
 
-						"fqdn": {
```

Comment:
By moving this out we will break existing terraform code. Could we mark this as deprecated to be removed in a future release and properly set it?